dist/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <meta charset="utf-8">
+   <meta name="viewport" content="width=device-width, initial-scale=1">
+
+   <title>Clock</title>
+   <link rel="stylesheet" href="app.css">
+</head>
+<body class="content" onload="setInterval('updateClock()', 200);">
```

Comment:
@aliaborroho where is this updateClock and `onload` attribute from? We have not covered this in class and it is a bad practice to mix Javascript in the HTML.